android/app/src/main/res/layout/max_widget_layout.xml

```diff
@@ -0,0 +1,427 @@
+                    android:background="#F6F6F7"
+                    android:gravity="center"
+                    tools:text="+12°" />
+
+            </RelativeLayout>
+
+            <RelativeLayout
+                android:id="@+id/weatherLayout4"
+                android:layout_width="wrap_content"
+                android:layout_height="80dp"
+                android:layout_marginEnd="8dp"
+                android:layout_toEndOf="@id/weatherLayout3"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/timeTextView4"
+                    style="@style/time"
+                    android:layout_width="wrap_content"
+                    android:layout_height="20dp"
+                    android:layout_alignStart="@id/weatherIconImageView4"
+                    android:layout_alignEnd="@id/weatherIconImageView4"
+                    android:background="#F6F6F7"
+                    android:gravity="center"
+                    tools:text="13:00" />
+
+                <ImageView
+                    android:id="@+id/weatherIconImageView4"
+                    android:layout_width="45dp"
+                    android:layout_height="45dp"
+                    android:layout_above="@id/temperatureTextView4"
+                    android:layout_below="@id/timeTextView4"
+                    android:layout_gravity="center"
+                    android:contentDescription="TODO"
+                    tools:src="@drawable/s32_light" />
+
+                <TextView
+                    android:id="@+id/temperatureTextView4"
+                    style="@style/temperatureSmall"
+                    android:layout_width="wrap_content"
+                    android:layout_height="20dp"
+                    android:layout_alignStart="@id/weatherIconImageView4"
+                    android:layout_alignEnd="@id/weatherIconImageView4"
+                    android:layout_alignParentBottom="true"
+                    android:background="#F6F6F7"
+                    android:gravity="center"
+                    tools:text="+12°" />
+
+            </RelativeLayout>
+
+        </RelativeLayout>
+
+        <TextView
+            android:id="@+id/crisisTextView"
+            style="@style/crisis"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            android:ellipsize="end"
+            android:maxLines="2"
+            android:paddingStart="6dp"
+            android:paddingTop="0dp"
+            android:paddingEnd="6dp"
+            android:paddingBottom="0dp"
+            android:visibility="gone"
+            tools:text="Viranomaistiedote: 13.11.2024 klo: 11.00: Alueella Lauttasaaren ranni..."
+            tools:visibility="visible" />
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageView
```

Comment:
FMI logo is not displayed in my device

Reply:
OK. It is marked as hidden.
So it is needed here?

I need to get the logo as a new SVG (which did not work earlier with Android Studio import) or I need several PNG versions.

Reply:
Yes, it is visible in max-widget according to Figma. You can find logo (light/dark) from here

https://github.com/fmidev/weather-app/pull/716/files

and later today in main-branch.

Reply:
This SVG file cannot be used in Android. I get these errors:
<img width="435" alt="Screenshot 2024-12-02 at 13 04 50" src="https://github.com/user-attachments/assets/0dc21255-0f53-448b-95d3-9b9a19e43e91">
ERROR @ line 22: <image> is not supported
ERROR @ line 41: <image> is not supported

I would need multiple PNG files as alternative:
https://developer.android.com/training/multiscreen/screendensities#TaskProvideAltBmp
But I do not know how to get them.

I am using a single PNG file exported from Figma.
But that is not enough.

Please create a ticket about this.

Reply:
Done